src/java.base/share/classes/java/lang/Thread.java

```diff
@@ -506,14 +506,14 @@ public static void sleep(long millis) throws InterruptedException {
             if (currentThread() instanceof VirtualThread vthread) {
                 vthread.sleepNanos(nanos);
             } else {
-                sleep0(millis);
+                sleep0(nanos);
             }
         } finally {
             afterSleep(event);
         }
     }
 
-    private static native void sleep0(long millis) throws InterruptedException;
+    private static native void sleep0(long nanos) throws InterruptedException;
```

Comment:
Maybe name this `sleepNanos0`?

Reply:
No, I don't think so. The name of this function is not relevant, as nothing else is calling it.